crawl4ai/__init__.py

```
@@ -24,9 +24,19 @@
     JsonCssExtractionStrategy,
     JsonXPathExtractionStrategy,
     JsonLxmlExtractionStrategy,
-    RegexExtractionStrategy
+    JsonLxmlExtractionStrategy_naive,
+    RegexExtractionStrategy,
```

Ruff (0.8.2)

`.extraction_strategy.RegexExtractionStrategy` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)